websites/M/Mahara-Tech/metadata.json

```
@@ -0,0 +1,40 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"id": "605894319408283678",
+		"name": "eljooker"
+	},
+	"service": "Mahara-Tech",
+	"description": {
+		"en": "information technology learning platform"
```

There may be a better description.
```suggestion
		"en": "Maharatech is an online learning platform, offering high-quality courses authored by Information Technology Institute ITI"
```
```suggestion
		"en": "The online learning hub for all passionate learners"
```